recipes/interpret-core/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "interpret-core" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/interpretml/interpret.git
+  git_rev: "v{{ version }}"
+  git_depth: 1
+
+build:
+  skip: true  # [py>=39 or py2k or py<37]
+  skip: true  # [win or osx]
+  number: 0
+  script: cp -r shared python/interpret-core/symbolic/shared
+  script: "npm install --package-lock-only --ignore-scripts && npx force-resolutions"
+  script: cd python/interpret-core && {{ PYTHON }} setup.py build && {{ PYTHON }} setup.py install
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - gxx  # [linux]
```

Comment:
When I remove gxx this is the error I get
$SRC_DIR/python/interpret-core/symbolic/build.sh: line 70: gcc: command not found

Reply:
Then there is a hardcoded `gcc` in that script. You will need to patch/fix that somehow otherwise, the build won't pick up the correct compiler. Only the compiler executables with the triplet `<arch>-<system>-gcc` are the ones that work correctly on conda-forge.

Reply:
Thanks for all the comments @xhochy 
I dont see anything obvious in the setup.py
https://github.com/interpretml/interpret/blob/v0.3.0/python/interpret-core/setup.py
Is adding gcc in meta.yaml a blocker to packaging interpret-core, if so how would I patch on conda-forge to make this work